src/server.js

```diff
@@ -1,51 +1,16 @@
 import express from "express";
-import handlebars from "handlebars";
-import fs from "fs";
+import {engine} from "express-handlebars"
 import presentations from "./db/presentations.json" with { type: "json" };
 import searchPresentations from "./utils/search.js";
+import * as path from "node:path"
+import {fileURLToPath} from "node:url"
+const __dirname = path.dirname(fileURLToPath(import.meta.url))
 
 const app = express();
+app.engine(".hbs", engine({extname: ".hbs"}))
```

Comment:
Use `.ebs` as engine and extension so that we can preserve our existing html filenames